src/validators.rs

```diff
@@ -6,6 +6,7 @@ use regex::Regex;
 use url::Url;
 
 const TITLE_MAX_LEN: usize = 44;
+const DESCRIPTION_MAX_LEN: usize = 125;
```

Comment:
Where does the 125 come from?

Why not old-school tweet-size? :)
```suggestion
const DESCRIPTION_MAX_LEN: usize = 144;
```

Reply:
Err meant 140, but 144 is fun too.

Reply:
That works too :)